acesso-io/unico-check-ios
# Feat/remove facetec binary

Why?
Allow 1.0.0 MAJOR release

Changes
remove facetec binary

How to test
`xcodebuild run test`

Preview
n/a

Issues
https://github.com/acesso-io/squad-check-mobile/issues/309
